### ISSUE TITLE
Unity 2017.1.3 don't support GetFrameworksBuildPhaseByTarget yet

### ISSUE THREAD

**Issue by havudk** [OneSignal/OneSignal-Unity-SDK#136]
**Description:**

I using unity 2017.1.3. When I switch to iOS platform this error come out:
```
Assets/OneSignal/Editor/PostProcessBuildPlayer_iOS.cs(156,49): error CS1061: 
Type `UnityEditor.iOS.Xcode.PBXProject' does not contain a definition for 
`GetFrameworksBuildPhaseByTarget' and no extension method `GetFrameworksBuildPhaseByTarget' 
of type `UnityEditor.iOS.Xcode.PBXProject' could be found. 
Are you missing an assembly reference?
```
In `PostProcessBuildPlayer_iOS.cs` I saw in line 126 `#if UNITY_2017_1_OR_NEWER && !UNITY_CLOUD_BUILD`
But when I research in Unity 2017.1 document It seem not support this function yet
https://docs.unity3d.com/2017.1/Documentation/ScriptReference/iOS.Xcode.PBXProject.html
Just from Unity 2017.2 It began support this function
https://docs.unity3d.com/2017.2/Documentation/ScriptReference/iOS.Xcode.PBXProject.html

Please update code or Correct me if I'm wrong.
Thank you very much.

**Environment**

1. OneSignal version: Unity-SDK-2.7.4
2. Unity 2017.1.3f1
3. iOS

**Comment by Nightsd01** [comment]
@havudk Thanks for reporting this! I've updated our iOS build script to account for this in #138

**Comment by Nightsd01** [comment]
Fixed the issue in #138, a new update should be released within the next week or so. In the mean time feel free to use master, it's a small fix.
